third_party/txt/src/txt/platform_linux.cc

```diff
@@ -13,6 +13,9 @@
 namespace txt {
 
 std::vector<std::string> GetDefaultFontFamilies() {
+#ifdef FLUTTER_USE_FONTCONFIG
```

Comment:
IMHO, if we fix this issue properly, if so, the previously committed code should be revert.
- 6e859d7ce (tag: 6e859d7) Create fallback font manager to solve performance drops (#78)
- 28dbd0208 (plugin_registrar) Fix font breaking issues (#69)

and I think we should only have one to two default fonts such as ```SamsungOneUI``` or ```TizenDefaultFont``` like other platform implementations

Reply:
I agree with you. 
If this patch lands, can you do it next time? :smile:

Reply:
Ok